### DISCUSSION TITLE
Vilken webbläsare för Apple-datorer?

### DISCUSSION THREAD

**Discussion by nikbrg62** [stefansundin/privatkopiera#235] [General]
Hej, funkar Privatkopiera bäst med Chrome eller Firefox i Apple-datorer?

**Comment by stefansundin** [comment]
Det fungerar precis lika bra i båda. 😄

**Reply by nikbrg62** [reply]
Hej Stefan... Jag är väldigt mycket nybörjare i det här, så det tog ett tag att fatta att privatkopiera är en application till själva webbläsaren. När jag nu försöker tillämpa privatkopiera på ett program på svtplay, får jag instruktionen att ladda ner FFmpeg för MacOS. När jag laddat ner tillägget och försöker öppna det får jag följande meddelande: "”ffmpeg” kan inte öppnas eftersom det kommer från en oidentifierad utvecklare. macOS kan inte verifiera att den här appen är fri från sabotageprogram."

Hur kommer jag vidare? Tack på förhand! :-)

**Reply by nikbrg62** [reply]
Tacksam för svar relativt omgående, då den dokumentär jag önskar ladda ner snart kommer plockas bort från SvTPlay...

**Reply by stefansundin** [reply]
Följ den här guiden: https://www.macworld.com/article/672947/how-to-open-a-mac-app-from-an-unidentified-developer.html

**Comment by nikbrg62** [comment]
Hej och tack igen!

Då har jag kunnat öppna ffmpeg och jag kan kopiera kommando.

MEN... sen kommer jag till punkt 5 i din instruktion till Privatkopiera (
https://stefansundin.github.io/privatkopiera/).

Där ska jag klistra in kommandot i en kommandotolk, som jag inte har
hanterat förut. Men det enda exemplet på kommandotolk är för Windows...

Så... Återigen tacksam för vägledning i nästa steg...

Allt gott / Nikolas

**Reply by stefansundin** [reply]
Hej. I macOS heter kommandotolkprogrammet "Terminal".. i alla fall på engelska macOS, vet inte om den heter annat på svenska macOS. Du kan hitta den i Launchpad (vet inte om det heter annorlunda på svenska heller).

**Reply by nikbrg62** [reply]
Hej och tack. Nu är det nära hoppas jag, eftersom filmen jag vill ladda ner tas bort 26 februari. Jag är inne i Terminal och försöker klistra in kommandot. Då säger Terminal: "ffmpeg: command not found".  Jag försöker, enligt dina instruktioner då följande "Om kommandotolken inte hittar ffmpeg så kan du behöva konfigurera sökvägen i tilläggets inställningar." Jag försöker öppna ffmpeg då jag tror att jag kommer åt inställningar där, men istället är jag tillbaka i Terminal (som på något sätt har blivit förval för att öppna ffmpeg vilket indikerar att de är kopplade...). Om jag försöker gå in på inställningar i Terminal fattar jag inget...

**Reply by stefansundin** [reply]
Hej. Med "tilläggets inställningar" menas Privatkopiera's inställningar. Kanske borde uppdatera den meningen så att den säger "Privatkopiera's inställningar" istället.

Vad det åstadkommer är egentligen bara att istället för att det står `ffmpeg` i början av kommandot så kanske det står hela sökvägen till `ffmpeg` filen, typ `/Users/nikbrg62/Downloads/ffmpeg`. Men du måste veta vart ffmpeg ligger på din hårddisk.

Om din `ffmpeg` fil ligger i din `Downloads` mapp så kan du behöva köra `./Downloads/ffmpeg` istället för bara `ffmpeg`. Om du vet hur man navigerar runt i terminalen med cd kommandot, då kan du först navigera till samma mapp som `ffmpeg` filen befinner sig i, och sedan köra `./ffmpeg`.

Lite komplicerat att göra första gången men sen borde det klicka.

**Reply by nikbrg62** [reply]
Tack, ja nu verkade det ju rassla på... Då är nästa fråga: Vart hamnade filmen? :-)

**Reply by stefansundin** [reply]
Den borde nog ha hamnat i din hem-mapp. Om du har svårt att hitta den så öppna Finder och använd "Go"-menyn.

![Screenshot 2025-02-22 at 08 18 41](https://github.com/user-attachments/assets/e918b97d-832f-48cf-a280-56ae398f80ed)

Om du använde `cd` för att navigera till en annan mapp så hamnar filen i den mappen.

**Reply by nikbrg62** [reply]
Yes nu hittade jag den, stort tack! Skulle gärna donera något, men har inte tillgång till något av de angivna sätten i nuläget...

**Comment by nikbrg62** [comment]
Hm... ska kanske "svara alla?" Skickar igen...

Hej och tack igen!

Då har jag kunnat öppna ffmpeg och jag kan kopiera kommando.

MEN... sen kommer jag till punkt 5 i din instruktion till Privatkopiera (
https://stefansundin.github.io/privatkopiera/).

Där ska jag klistra in kommandot i en kommandotolk, som jag inte har
hanterat förut. Men det enda exemplet på kommandotolk är för Windows...

Så... Återigen tacksam för vägledning i nästa steg...

Allt gott / Nikolas

Nikolas Berg och Ingrid Berg
Ekopedagoger, föreläsare, författare
Medskaparna - lärande för hållbarhet
***@***.*** <http://www.medskaparna.se>, 070-7544840


Den ons 12 feb. 2025 kl 21:38 skrev Stefan Sundin ***@***.***
